demo/admin/src/products/future/generated/ProductsGrid.tsx

```diff
@@ -87,7 +92,32 @@ function ProductsGridToolbar({ toolbarAction }: { toolbarAction?: React.ReactNod
                 <GridFilterButton />
             </ToolbarItem>
             <ToolbarFillSpace />
+            <ToolbarItem>
```

Comment:
We should use the CrudMoreActionsComponent for this

Reply:
Good point 👍🏼

Reply:
Done in e32e4e0e1b2f8c9187c9c53374fff8782d14aff8